doxia-modules/doxia-module-xhtml/src/main/java/org/apache/maven/doxia/module/xhtml/XhtmlParser.java

```diff
@@ -269,11 +269,11 @@ private void processMacro( String text, Sink sink )
         }
         catch ( MacroExecutionException e )
         {
-            throw new XmlPullParserException( "Unable to execute macro in the document: " + macroName );
+            throw new XmlPullParserException( "Unable to execute macro in the document: " + macroName , null, e );
```

Comment:
can you provide an example of failing macro for me to test in real conditions how the failure renders without and with the update, please?

Reply:
@hboutemy  See my comment above :)